lib/pmdtester/parsers/options.rb

```diff
@@ -111,20 +116,17 @@ def check_options
     def check_local_options
       check_option(LOCAL, 'base branch name', @base_branch)
       check_option(LOCAL, 'base branch config path', @base_config) unless @auto_config_flag
-      check_option(LOCAL, 'patch branch name', @patch_branch)
       check_option(LOCAL, 'patch branch config path', @patch_config) unless @auto_config_flag
       check_option(LOCAL, 'list of projects file path', @project_list)
     end
 
     def check_single_options
-      check_option(SINGLE, 'patch branch name', @patch_branch)
```

Comment:
I think, we still need the patch_branch option, don't we? Ah... it's checked already in `check_common_options`...